highlevel/robot/controller/motion/motion.py

```diff
@@ -41,6 +40,7 @@ class MotionStatus:
     ramp_angle: Radian
     is_arrived: bool
     is_blocked: bool
+    is_enabled: bool
```

Comment:
what does it mean for a motion to be enabled? Maybe you can add a comment tehre

Reply:
It wasn't meant to be kept, but it will be usefull in a future version if we need to have the robot's wheels free (for match preparation).
We can remove it.